public/doc/openapi-3-v1.0.0.json

```diff
@@ -1793,6 +1796,9 @@
         "parameters": [
           {
             "$ref": "#/components/parameters/PortfolioItemID"
+          },
+          {
+            "$ref": "#/components/parameters/QueryFormat"
```

Comment:
@lindgrenj6 wouldn't this change the response object which can have different types based on the query format in the request?

Reply:
As far as I know I can't change the response to be multiple just due to the fact that we can't use `oneOf` in our spec due to the python client not enjoying it. 

I think we should keep it the way it is since the only time the output would change is if they add the query parameter. Suggestions?

Reply:
@mkanoor do you have any thoughts or suggestions?

Reply:
@lindgrenj6 Since this is only being used for the UI can we ask what format they need it in and set that in the response body and not give them a format option. Or have a different request for this other format. If the response is going to flex based on what the input format is it won't bode well for the openapi.json which expects the response to be well defined.

Reply:
@mkanoor technically we're already not representing it right in the first place - since the response could be any one of:
- `image/png`
- `image/jpeg`
- `image/svg+xml`

and we'd be adding `application/json`

So we're already in a pickle since we can't use `oneOf` in the response for the separate image types. @Hyperkid123 also said he would prefer a query parameter rather than building out a new endpoint.